event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/event/EventJdbcRepository.java

```diff
@@ -27,7 +27,8 @@
  * JDBC based repository for event log records.
  */
 @ApplicationScoped
-public class EventJdbcRepository extends AbstractJdbcRepository<Event> implements PaginationCapableRepository<Event> {
+public class EventJdbcRepository extends AbstractJdbcRepository<Event>
+        implements PaginationCapableRepository<Event> {
```

Comment:
PaginaionCapableRepository is not required.

Reply:
Removed

Reply:
Removed